### PR TITLE
Add pagination to Search result

### PR DESCRIPTION
**Pull request by xiaojun207**
Add pagination to Search result,

### CODE REVIEW

**Comment by hengfeiyang** [comment]
It's has problem of `from: 1`, default should `from: 0`

**Comment by xiaojun207** [comment]
> It's has problem of `from: 1`, default should `from: 0`

Ok，I has set `from` default from 1 to 0.

File: web/src/components/search/SearchResult.vue [hunk 1]
```diff
@@ -4,14 +4,16 @@
       <q-table
         ref="searchTable"
         v-model:expanded="searchResult._source"
+        v-model:pagination="pagination"
         data-cy="search-result-area"
         :rows="searchResult"
         :columns="resultColumns"
         :loading="searchLoading"
-        :pagination="pagination"
+        :rows-per-page-options="[5, 10, 20, 50, 100, 500, 1000, 5000, 10000]"
```
Comment (hengfeiyang) [inline web/src/components/search/SearchResult.vue:12]: I think page options just have `[5, 10, 20, 50, 100]` is ok, it looks no need very large page. And you should use a variable to set it.

**Review by hengfeiyang — COMMENTED** [review]
And the column sort is not work.

**Comment by hengfeiyang** [comment]
Do you think we should limit the max returns, i remember the limit is 5000 in ES.

**Comment by xiaojun207** [comment]
> Do you think we should limit the max returns, i remember the limit is 5000 in ES.

This depends on the impact of paging on performance. Will the recording performance of more than 5000 decline rapidly?

If the server has no paging performance problems, there is no need to limit the maximum return.

**Comment by hengfeiyang** [comment]
> > Do you think we should limit the max returns, i remember the limit is 5000 in ES.
> 
> This depends on the impact of paging on performance. Will the recording performance of more than 5000 decline rapidly?
> 
> If the server has no paging performance problems, there is no need to limit the maximum return.

For now, yes, the server need skip `FROM` then returns `SIZE`

**Comment by hengfeiyang** [comment]
@xiaojun207  We discussed about this PR, we think we will not accept it, because the PR need load data from server every request, but for most user they just need the topN data, just one request is better than many times request, it will reduce server resource.

**Pull request closed without merging**